src/services/region-loader.ts

```diff
@@ -0,0 +1,6 @@
+export const getRegionId = () =>
+  localStorage.getItem('region') || process.env.REACT_APP_INITIAL_REGION || 'ca'
```

Comment:
this could've been static if we didn't support switching

Reply:
more or less removed it. Available only on localhost